include/gdsync/core.h

```diff
@@ -114,10 +163,11 @@ int gds_stream_queue_send(CUstream stream, struct gds_qp *qp, gds_send_wr *p_ewr
 // batched submission APIs
 
 typedef enum gds_memory_type {
-        GDS_MEMORY_GPU  = 1, /*< use this flag for both cudaMalloc/cuMemAlloc and cudaMallocHost/cuMemHostAlloc */
+        /* use this flag for both cudaMalloc/cuMemAlloc and cudaMallocHost/cuMemHostAlloc */
```

Comment:
please move this comment back inline for doxygen, one day we will enable it

Reply:
this is still here

Reply:
Reverted